### ISSUE TITLE
Validation/Test Performance Gap in Tutorial Notebook

### ISSUE THREAD

**Issue by PrettyMeng** [TradeMaster-NTU/TradeMaster#125]
Hi, thanks for making your great work public.

I'm a bit confused about the huge test/validation performance gap in some of your tutorials.  In particular, the tutorial for DeepScalper achieves a negative return, but a +200% return on the test set, which does not make sense to me. Similarly, in the notebook for double DQN, the validation return is also negative, while achieving a positive return on the test set. 

Intuitively, it's hard to believe that we are able to achieve a positive return in the future if we cannot even achieve a positive return on the validation set.

Am I missing something here?

**Comment by sunshuo01** [comment]
Hi,

Thanks for your interest on our project. I think your concern is reasonable. The big performance gap between validation and test might be caused by the following reasons: 1) Comparing to other RL domains, financial market is regarded with low signal-to-noise ratio and significant distribution shift among time. As a result, a relatively large gap between validation and test is Ok in this domain. I do agree that the current gap seems too much and will try to fix it later. 2) the dataset we offer here is quite small and might not be enough for RL algorithms to fully coverage. We provide them as an example here and the performance will be more stable if more data is available based on my experience. 3) In tutorials, we only train the agents for a few episodes to save time. Training longer might help in practice.
